newIDE/app/src/ObjectsList/index.js

```diff
@@ -228,7 +231,8 @@ const ObjectsList = React.forwardRef<Props, ObjectsListInterface>(
         if (onEditObject) {
           onEditObject(object);
           onObjectCreated(object);
-          onObjectSelected(name);
+          // Adding an object is always (at the moment) going to the scene (layout), and not to the project (as a global object) so the context is not global.
```

Comment:
Could you put this comment on multiple lines so that it's easy to read?
The number of characters before the formatter automatically wraps the line is 80.
You can use something similar (no need to be perfectly accurate)

Reply:
Hi,
Done and pushed.